src/util/log.rs

```diff
@@ -61,6 +64,37 @@ fn print_msg_header(mut rd: &mut dyn RecordDecorator, record: &Record) -> io::Re
     Ok(count_rd.count() != 0)
 }
 
+fn pretty_print_msg_header(rd: &mut dyn RecordDecorator, record: &Record, debug: bool) -> io::Result<bool> {
+    rd.start_timestamp()?;
+    let now: DateTime<Utc> = Utc::now();
+    write!(
+        rd, "\x1b[0;90m{}", now.format("%b %e %T"))?;
```

Comment:
The pretty print looks great. I don't think this will work on Windows console by default, a win32 API has to be called at runtime to enable it, otherwise I think it will just print the escape codes.

Reply:
Please remove terminal coloration altogether, or gate it with a feature flag or an environment variable.  It should be off by default.

First, when you have multiple threads writing to the same terminal device file (i.e. the pty behind your console), you can easily break coloration if one thread yields to another thread without first writing the escape sequence to disable coloration.  This can leave your console output looking corrupted.

Second, you're not using the `isatty(2)` check here, so if I redirect logging to a file, it will be filled with these escape sequences.

Reply:
As explained in the introduction, this is off by default - gated with `STACKS_LOG_PP`

Reply:
Ah, just saw that.  But, you'll still want want to guard this with `isatty(2)`.  In general, piping or redirecting the output of a program ought to disable coloration, since you (almost) never want to see the escape codes as actual text.

Reply:
Yeah definitely will, thanks for the tip

Reply:
If we perform the `isatty` check, can we also enable the Win32 API for coloring? Would be super helpful. I for one would be running a Stacks 2.0 on Windows with pretty print enabled.

Reply:
Addressed with latest. @zone117x I don't have what is needed for testing a windows setup, feel free to push a commit / follow-up with something tested :)